docs/using-airbyte/pyairbyte/getting-started.mdx

```diff
@@ -70,3 +70,11 @@ PyAirbyte is a python library that can be run in any context that supports Pytho
 The following connectors are available:
 
 <AirbyteLibConnectors />
+
+## LangChain integration
+
+For those interested in using PyAirbyte to drive your LLM use cases, we provide two ways to integrate with LangChain:
+
+* **LangChain native integration**: This approach requires you to utilize the langchain-airbyte integration package. Refer to [LangChain Docs](https://python.langchain.com/docs/integrations/document_loaders/airbyte) or watch this [YouTube video](https://www.youtube.com/watch?v=zQU_1sCLSMU) to get started.
```

Comment:
Minor nit. Surrounding the library name with backticks will make it more clear this is an exact package name. 

```suggestion
* **LangChain native integration**: This approach requires you to utilize the `langchain-airbyte` integration package. Refer to [LangChain Docs](https://python.langchain.com/docs/integrations/document_loaders/airbyte) or watch this [YouTube video](https://www.youtube.com/watch?v=zQU_1sCLSMU) to get started.
```

Reply:
will do that.

Reply:
done.